bin/opencanary.tac

```diff
@@ -22,7 +22,7 @@ from opencanary.modules.git import CanaryGit
 from opencanary.modules.redis import CanaryRedis
 from opencanary.modules.tcpbanner import CanaryTCPBanner
 from opencanary.modules.rdp import CanaryRDP
-
+from opencanary.modules.llmnr import CanaryLLMNR
```

Comment:
Could we copy how we handle the imports for the the SNMP module here? [SNMP import](https://github.com/thinkst/opencanary/blob/5b6dbd01b9cc404f8d5b96f8e9be8b620189078a/bin/opencanary.tac#L59)

Reading the docs it looks like we have a weak dependency on Scapy. It doesn't have to be installed. So this change will just make sure folks pulling a newer version won't have the thing falling over if they didn't have Scapy installed originally (and might not use this new service).